pkgs/top-level/python-packages.nix

```
@@ -5388,6 +5388,10 @@ in {
 
   pysensors = callPackage ../development/python-modules/pysensors { };
 
+  python_toolbox = callPackage ../development/python-modules/python_toolbox { };
```

```suggestion
  python-toolbox = callPackage ../development/python-modules/python-toolbox { };
```
See https://nixos.org/nixpkgs/manual/#contributing.